src/graphql/makeGraphQLRequest.ts

```diff
@@ -27,15 +29,15 @@ export async function makeGraphQLRequest<T, V = Variables>(
       const result = await client.request(document, variables, requestHeaders);
       return result;
     } catch (err) {
-      if (retryCount > MAX_RETRIES) {
+      if (retryCount >= maxRequests) {
```

Comment:
So it's technically 5 retries now?

Reply:
i think its the same, i just made it log out starting from 1 instead of 0